apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/features/TestFeatures.java

```diff
@@ -42,13 +45,31 @@ public static void initialize(Activity parentActivity) {
         sTestFeatureModels.add(new TestFeatureTitle(R.string.crashes_title));
         sTestFeatureModels.add(new TestFeature(R.string.title_crashes, R.string.description_crashes, CrashActivity.class));
         sTestFeatureModels.add(new TestFeature(R.string.title_error, R.string.description_error, ManagedErrorActivity.class));
+        sTestFeatureModels.add(new TestFeature(R.string.title_had_memory_warning, hadMemoryWarning() ? R.string.description_had_memory_warning : R.string.description_didnt_have_memory_warning, new View.OnClickListener() {
```

Comment:
Can we pass null here?

Reply:
No. The listener is final. I wanted to change that but didn't

Reply:
final variables can be null, not sure what is the problem here with null

Reply:
Just need to check for null when using it

Reply:
It either complains it's not initialized or can't find the matching constructor

Reply:
Had to add a cast which I don't like